src/main/java/Robots/samples/ObstacleAvoidRobot.java

```diff
@@ -54,6 +55,33 @@ public void loop() throws Exception {
             } else {
                 motion.move(defaultMoveSpeed, defaultMoveSpeed, 1000);
             }
+        }*/
+
+        if(state == robotState.RUN){
+            double dist = distSensor.getDistance();
+            System.out.println("dist  3 " +distSensor.getDistance());
+
+
+            if(dist<=10){
+                if(dist<=1){
+                    motion.rotateDegree((int) (defaultMoveSpeed * 0.75), 91);
+                    if(distSensor.getDistance()<=1){
+                        motion.rotateDegree((int) (defaultMoveSpeed * 0.75), -91);
+                        motion.rotateDegree((int) (defaultMoveSpeed * 0.75), -91);
```

Comment:
If you need 180 deg rotation, you can give it in one function call

Reply:
If we want -180 degree rotation we want to give it as -182degree.But u have set limitation as degree < -180.0F || degree > 180.0F.Therefore I did like that.